app/themes/map/ui/styles/theme.less

```diff
@@ -0,0 +1,181 @@
+}
+
+#map-controls {
+  position: fixed;
+  top: 65px;
+  left: 15px;
+  width: 60%;
+  max-width: 300px;
+}
+
+#map-controls {
+  transition-opacity: 0.6;
+  transition: opacity ease 0.3s;
+  transition-delay: 0.3s;
+
+  .panel-default-shrink { background-color: #FEFEFE; position: relative; max-height: 40px; margin-top: -50px; transition: all ease 0.3s; overflow: hidden; transition-delay: 0.3s; }
+  .panel-default { background-color: #FEFEFE; margin-top: 15px; }
+
+  .panel-default:first-child { z-index: 503; }
+  .panel-default:nth-child(2) { z-index: 502 }
+  .panel-default:nth-child(3) { z-index: 501 }
+
+  &:hover, &.fresh {
+    opacity: 1.0;
+
+    .panel-default { max-height: 500px; margin-top: 15px; }
+  }
+
+
+  select, li { font-size: 0.975em; }
+
+}
+
+.map-info-window {
+
+  .heading {
+    position: relative;
+    background-color: #f75000;
+    color: #FFF;
+
+    .ranking {
+      display: inline-block;
+      left: 5px;
+      padding: 4px;
+      background-color: #971a25;
+    }
+    .window-title {
+      display: inline-block;
+      margin: 0 0 0 5px;
+      padding: 2px 0 0 0;
+    }
+  }
+
+  .photo {
+    display: inline-block; vertical-align: top; width: 45%;
+
+    img { width: 100%; height: auto; margin-top: 10px; }
+  }
+  .details { display: inline-block; width: 50%; margin-left: 1%}
+
+  addr { display: block; font-weight: bold; padding: 5px 0; }
+
+  .dl-horizontal {
+    dt { width: 25%; text-align: left; }
+    dd { margin-left: 20px; }
+  }
+}
+
+#powered-by {
+  display: block;
+  position: absolute;
+  bottom: 30px;
+  left: 7px;
+  background-image: url('../images/MarkLogic-Powered-By.png');
+  background-repeat: no-repeat;
+  background-size: contain;
+  width: 23%;
+  height: 23%;
+  max-width: 99px;
+  max-height: 34px;
+  z-index: 500;
+}
+
+#mobile-info-window { display: none; }
+
+#hamburger { display: none; }
+
+@media(max-width: 992px) {
+
+  #logo { left: 25px; }
+
+  #hamburger {
+    display: block;
+    position: absolute;
+    top: 24px;
+    left: 10px;
+    font-size: 1.5em;
+  }
+
+  #map-controls {
+
+    .panel-default:nth-child(2) {
+      border-top: 0;
+      margin-top: -25px;
+    }
+  }
+
+  #mobile-info-window {
+    position: absolute;
+    left: 0;
+    bottom: 0;
+    width: 100%;
+    height: 0;
+    display: block;
+
+    .info-window {
+      position: relative;
+      margin-top: -250px;
+      height: 250px;
+      background-color: #FFF;
+      width: 100%;
+      z-index: 600;
+      overflow: auto;
+      display: none;
+
+      &.isShown {
+        display: block;
+      }
+
+      .photo {
+        position: absolute;
+        right: 15px;
+        top: 65px;
+        z-index: 599;
+      }
+
+      .heading {
+        position: fixed;
+        bottom: 222px;
+        width: 100%;
+        z-index: 600;
+      }
+
+      .window-content {
+        padding-top: 31px;
+
+        .details { width: 100%; }
+        .dl-horizontal dt { width: 50%; }
+      }
+    }
+  }
+}
```

Comment:
Add this selector at the end for the new copyright.
` p.map-copyright {
  color: #333;
  font-size: .8em;
  margin-left: 100px;
  position: fixed;
  bottom: -8px;
}`